.github/workflows/github-actions-cron-test-installer.yml

```diff
@@ -0,0 +1,98 @@
+name: Test DependencyInstaller script
+on:
+  schedule:
+  - cron: "0 8 * * SUN"
+  push:
+    paths:
+      - 'etc/DependencyInstaller.sh'
+      - '.github/workflows/github-actions-cron-test-installer.yml'
+      - 'build_openroad.sh'
+      - 'setup_env.sh'
+  pull_request:
+    paths:
+      - 'etc/DependencyInstaller.sh'
+      - '.github/workflows/github-actions-cron-test-installer.yml'
+      - 'build_openroad.sh'
+      - 'setup_env.sh'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:     
+  testInstaller:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu:20.04", "ubuntu:22.04", "centos:centos7"]
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.os }}
+    steps:
+      - name: Setup
+        run: |
+          if [ "${{ matrix.os }}" != "centos:centos7" ]; then 
+            apt-get update
+            apt-get -y install git
+          else
+            yum -y install https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm
+            yum -y install git
+          fi
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Checkout repository code with submodules
+        run: |
+          git config --global --add safe.directory '*'
+          git submodule update --init
```

Comment:
doesn't `submodule: recursive` works?

Reply:
it causes this error when it runs inside a container: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/actions/runs/3780081571/jobs/6425873507